pkg/repo/mysql/message.go

```diff
@@ -117,35 +118,44 @@ func (r *Repo) LoadMessages(ctx context.Context, serviceID uuid.UUID, opts commo
 
 	defer rows.Close()
 
-	messagesLookup := make(map[string][]model.Message)
+	messagesLookup := make(map[string]*model.Messages)
```

Comment:
What's the purpose of having pointers to model.Messages?

Reply:
That way modifying it here

https://github.com/translate-agent/translate/blob/ddd9e4757ba47001bbee9cd023d796b58b656ff2/pkg/repo/mysql/message.go#L145-L146

It also modifies in a lookup map